src/Spice86.Core/Emulator/InterruptHandlers/Input/Mouse/MouseInt33Handler.cs

```diff
@@ -157,6 +159,20 @@ public class MouseInt33Handler : InterruptHandler {
         _userCallbackSegment = newUserCallbackSegment;
     }
 
+    public void ShowMouseCursor() {
+        if (_logger.IsEnabled(Serilog.Events.LogEventLevel.Information)) {
+            _logger.Information("SHOW MOUSE CURSOR (unimplemented!)");
+        }
+        // @TODO: show mouse cursor
+    }
+
+    public void HideMouseCursor() {
+        if (_logger.IsEnabled(Serilog.Events.LogEventLevel.Information)) {
+            _logger.Information("HIDE MOUSE CURSOR (unimplemented!)");
+        }
+        // @TODO: hide mouse cursor
```

Comment:
Optional, as it can be picked up later: implement this TODO ?

Reply:
I could not figure out how to hide the mouse pointer in AvaloniaUI :(

Reply:
The good news is, is that the emulated cursor is not controlled by Avalonia.
If the UI is present (as Spice86 also features a headless mode), the system cursor is always shown in Spice86, on top of the emulated one. Unless set as hidden in the UI options.

But again, only the emulated one neeeds to be shown/hidden here.

This could be implemented later, anyway. Knowing that the host program wants to show/hide the mouse cursor is good enough to know what it's doing and replacing that with a higher level code.